### PR TITLE
feat: add 100th catchup summary

### PR DESCRIPTION
**Pull request by SirusCodes**

### CODE REVIEW

File: summary/sessions/100/content.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Anil Harwani shared how https://www.dpdk.org/[DPDK (Data plane development kit)] handles the networking in most mainframes.
+* Anil Harwani showed us some of lesser known Linux commands
+    ** https://man7.org/linux/man-pages/man1/lscpu.1.html[lscpu] - lscpu gathers CPU architecture information sysfs, /proc/cpuinfo and any applicable architecture-specific libraries (e.g. Librtas on PowerPC).
+    ** https://man7.org/linux/man-pages/man1/w.1.html[w] - w displays information about the users currently on the machine, and their processes
+    ** https://man7.org/linux/man-pages/man1/wall.1.html[wall] -  wall displays a message, or the contents of a file, or otherwise its standard input, on the terminals of all currently logged in users.
+* We got to know about https://www.techtarget.com/searchnetworking/definition/jumbo-frames[jumbo frames] by Anil Harwani, which makes transmission fasters as the frame size are bigger which in turn takes less time to process.
```
Comment (tusharnankani) [inline summary/sessions/100/content.adoc:15]: > jumbo frames by Anil Harwani

I read it as this as it was something by him^

Let's keep this consistent. Ex: the next 2 lines.
Anil Harwani spoke/enlightened us about...

File: summary/sessions/100/content.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Anil Harwani shared how a https://www.phoronix.com/news/Linux-AMD-Old-Chipset-WA[workaround in Linux Kernel was hurting modern AMD's performance].
+* Anil Harwani showed how https://www.geeksforgeeks.org/screen-command-in-linux-with-examples/[screen command] in Linux helps him in his workflow.
+    ** screen command in Linux provides the ability to launch and use multiple shell sessions from a single ssh session.
```
Comment (tusharnankani) [inline summary/sessions/100/content.adoc:18]: `screen`*

File: summary/sessions/100/content.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Date: 08-10-2022
+
+Duration: 9 hrs 48 mins
+
+==== Topics Discussed
+
+* We completed 100 consecutive CatchUps 🥳
+* Anil Harwani shared the https://github.com/esnet/iperf[github.com/esnet/iperf] repository which is a measurement of TCP, UDP and SCTP network bandwidth.
```
[the remark this answers is not in the document]
Comment (tusharnankani) [reply summary/sessions/100/content.adoc:8]: Like how it's done [here](https://github.com/OurTechCommunity/catchup/pull/95/files#diff-b8ff77c375c0ee33811b98592904f0d889f431b461e72cf118cca87642708fb7R19).

[the remark this answers is not in the document]
Comment (SirusCodes) [reply summary/sessions/100/content.adoc:8]: Ik but it doesn't matter. I have not followed this for any of the summaries I wrote 😢

File: summary/sessions/100/content.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* Anil Harwani explained how a packet travels from the network card to application. https://www.cs.dartmouth.edu/~sergey/netreads/path-of-packet/Network_stack.pdf["Path of a packet in the Linux kernel stack"] explains it in-depth.
+* Anil Harwani shared how https://www.dpdk.org/[DPDK (Data plane development kit)] handles the networking in most mainframes.
+* Anil Harwani showed us some of lesser known Linux commands
+    ** https://man7.org/linux/man-pages/man1/lscpu.1.html[lscpu] - lscpu gathers CPU architecture information sysfs, /proc/cpuinfo and any applicable architecture-specific libraries (e.g. Librtas on PowerPC).
+    ** https://man7.org/linux/man-pages/man1/w.1.html[w] - w displays information about the users currently on the machine, and their processes
```
[the remark this answers is not in the document]
Comment (SirusCodes) [reply summary/sessions/100/content.adoc:13]: I copied the line from the man file. But will change it.

File: summary/sessions/100/content.adoc (hunk 1 above)
Comment (SirusCodes) [reply summary/sessions/100/content.adoc:15]: Cool

File: summary/sessions/100/content.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (tusharnankani) [reply summary/sessions/100/content.adoc:8]: What?

**Pull request merged**